keyboards/mawaeg/sharkropad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2025 @mawaeg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    // clang-format off
+    /*
+     *             ┌────────────┐
+     *             │ RGB_MOD    │
+     * ┌───┬───┬───┼────────────┤
+     * │ 7 │ 8 │ 9 │ RGB_TOGGLE │
+     * ├───┼───┼───┼────────────┤
+     * │ 4 │ 5 │ 6 │ NUM        │
+     * ├───┼───┼───┼────────────┤
+     * │ 1 │ 2 │ 3 │ 0          │
+     * └───┴───┴───┴────────────┘
+     */
+        [0] = LAYOUT_numpad_4x4(
```

Comment:
There is already a numpad 4x4 layout, and this doesn't match.  `LAYOUT` would be better here. 
https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_4x4/keymap.c

```suggestion
        [0] = LAYOUT(
```